src/components/ThemeProvider/ThemeProviderUtils.ts

```diff
@@ -54,3 +54,24 @@ export const generateRandomAlphaString = (length = 6) => {
 
   return result;
 };
+
+const APP_THEME_SUFFIX = "-app-theme";
+
+export const getBodyThemeClassName = () => {
+  const classList = document.body.classList;
+  let bodyAppThemeClassName: string = null;
+  classList.forEach(className => {
+    if (className.endsWith(APP_THEME_SUFFIX)) {
+      bodyAppThemeClassName = className;
+    }
+  });
+  return bodyAppThemeClassName;
+};
+
+export const addThemeClassNameToBody = (theme: SystemTheme) => {
+  document.body.classList.add(`${theme}${APP_THEME_SUFFIX}`);
+};
+
+export const removeThemeClassNameFromBody = (theme: SystemTheme) => {
+  document.body.classList.remove(`${theme}${APP_THEME_SUFFIX}`);
+};
```

Comment:
I'd validate here that `theme` is a string and not an object for the possible prop renaming in [this PR](https://github.com/mondaycom/monday-ui-react-core/pull/1869)

Reply:
you can even provide a meaningful `console.error` to help understand what went wrong and why the theme isn't applied

Reply:
Aren't I'm safe with the type checking?